MAINTAINERS.yml

```diff
@@ -3443,9 +3447,10 @@ NXP Platforms (MPU):
     - soc/nxp/layerscape/
     - boards/nxp/ls1046ardb/
   files-regex:
-    - boards/nxp/m?imx[^(rt)].*/
+    - boards/nxp/.*m?imx[^(rt)].*/
```

Comment:
Can we remove `m?` i.e match it to `.*imx[^(rt)].*/`

Reply:
in the past there was some imx board that started with m, i dont know if they got renamed or something, but this doesnt need to change, it already doesnt match the rt ones